docs/tutorials/js-sdk-tutorial.md

```diff
@@ -0,0 +1,144 @@
+---
+Title: Datahub JS SDK Tutorial
+---
+
+## Introduction
+
+The [DataHub](http://datahub.io/) platform stores a lot of different datasets - which are packages of useful data alongside with the description (here is [the dataset specification](https://frictionlessdata.io/docs/data-package/)). The data, stored on the DataHub, has a nice structure, views and a description that help people to get insights.
+You can also store and share your own datasets on the DataHub
+
+As a programmer, you may want to automate the process of getting or storing the data. Also you may want to integrate your project and the DataHub.
+
+This tutorial will cover information you need to create the data driven project that will use the DataHub to get or store the data.
+
+Let's explore the tutorial together below.
+
+**Important notes:**
+- You need to use **Node version > 7.6**
+- When you see the **await** keyword you should wrap the code in the **async** function.
+
+
+## Install
+
+Add `datahub-client` package into your project:
+```shell
+npm install datahub-client --save
+```
+The `datahub-client` is a JS library to communicate with the DataHub and to create and validate datasets. It also includes the `data.js` package.
+`Data.js` provides `Dataset` and `File` classes, that represents a datapackage and a data-file.
+
+
+## Quick overview
+
+After loading 'datahub-client' into your program:
+`const datahub = require('datahub-client')` ,
+you can do things like:
+
+- login to DataHub
+- authenticate with the jwt token
+- push a dataset:
+- get the data from the DataHub:
+- init a dataset
+- validate a dataset
+- convert tabular data files into different formats
+
+Let's explore some of the `datahub-client` features more deeply below.
+
+
+## Push data
+
+### Prepare the test data and user credentials
+1. Here is a [Sample dataset on Github](https://github.com/datasets/finance-vix/archive/master.zip).  Please, download and unpack it.
```

Comment:
Again, this is not good Idea. Have you actually tried publishing `fininace-vix` got from datahub?

Reply:
This is a original dataset from the GITHUB